### PR TITLE
Refactor report errors for the APIv2 (part of #374)

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
* Add decorator for error subtitles
* Wrap null, undefined number, etc. with \<code\> in messages

/cc @inikulin

### CODE REVIEW

File: src/errors/test-run/templates.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default {
     `),
 
     [TYPE.actionStringArrayElementError]: err => markup(err, `
-        Elements of the <code>${err.argumentName}</code> argument are expected to be non-empty strings, but the element at index ${err.elementIndex} was ${err.actualValue}.
+        Elements of the <code>${err.argumentName}</code> argument are expected to be non-empty strings, but the element at index <code>${err.elementIndex}</code> was ${err.actualValue}.
```
Comment (inikulin) [inline src/errors/test-run/templates.js]: I think we should wrap `actualValue` here not index. BTW, it worth cheking if `actualValue` not wrapped elsewhere

Comment (AlexanderMoskovkin) [inline src/errors/test-run/templates.js]: The sub-task in the issue is `Wrap null, undefined number, etc. with <code> in messages`.
`${err.elementIndex}` is always number. Why we shouldn`t wrap it?

Comment (inikulin) [inline src/errors/test-run/templates.js]: OK, keep it as is

Comment (AlexanderMoskovkin) [inline src/errors/test-run/templates.js]: So, what about other changes. Tests are ok

**Comment by inikulin** [comment]
lgtm

**Pull request merged**